src/section-addition-and-subtraction-of-rational-expressions.ptx

```diff
@@ -33,17 +33,18 @@
 
 <!--TODO: Do something about "Town A" and "Town B"-->
         <example xml:id="example-boat-river-rational-function">
-            <p>Two towns are connected by a <m>12</m>-mile-long river, which flows from Town A to Town B at the velocity of <m>2</m> miles per hour. A boat will travel at a constant speed, <m>v</m> miles per hour, from Town A to Town B, and then back to Town A. Due to the river's flow, the boat's actual speed is <m>v+2</m> miles per hour from Town A to Town B, and <m>v-2</m> miles per hour from Town B back to Town A. If the boat driver plans to spend <m>8</m> hours for the whole trip, how fast should he/she drive the boat?</p>
+            <p>Two towns are connected by a <m>12</m>-mile-long river, which flows from Town A to Town B at a velocity of <m>2</m> miles per hour. A boat will travel at a constant speed, <m>v</m> miles per hour, from Town A to Town B, and then back to Town A. Due to the current of the river, the actual speed of travel is <m>v+2</m> miles per hour when going downstream from Town A to Town B, and <m>v-2</m> miles per hour going upstream back to Town A. If the boat driver plans to spend <m>8</m> hours for the whole trip, how fast should they drive the boat?</p>
 
-            <p>We need to review three formulas for constant-rate movement:<md>
+<!-- TODO: Add xref  -->
+            <p>We need to review three forms of the formula for movement at a constant rate:<md>
                 <mrow>d\amp=vt\amp v\amp=\frac{d}{t}\amp t\amp=\frac{d}{v}</mrow>
-            </md>where <m>d</m> stands for distance, <m>v</m> stands for speed, and <m>t</m> stands for time. According to the third formula, the time it takes the boat to travel from Town A to Town B is <m>\frac{12}{v+2}</m>, and <m>\frac{12}{v-2}</m> from Town B to Town A.</p>
+            </md>where <m>d</m> stands for distance, <m>v</m> stands for velocity, and <m>t</m> stands for time. According to the third form, the time it takes the boat to travel from Town A to Town B is <m>\frac{12}{v+2}</m>, and the time it takes to get back is <m>\frac{12}{v-2}</m>.</p>
```

Comment:
I think we should make this be about "speed". "velocity" conjures the vector meaning of that word, and you'd have to talk about negative velocities as it traveled in one direction.